src/Microsoft.AspNet.Mvc.Core/Formatters/IWrapperProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+    using System;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public interface IWrapperProvider
```

Comment:
This is a really bad name in the Mvc namespace.

These wrappers are intended for XML only, and should probably renamed